# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpChannel.cpp
# Commit: ca5fed1e3908
# Full Hash: ca5fed1e3908c850b24ccedd0e8762c3b2854171
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2023-03-16 12:57:15
# Description:
#   Bug 1822801 - socketTransport->GetEffectiveTRRMode may fail r=kershaw a=crash-fix
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D172788
# ==============================================================================

diff -r 7954e1671be3 -r ca5fed1e3908 netwerk/protocol/http/nsHttpChannel.cpp
--- a/netwerk/protocol/http/nsHttpChannel.cpp	Thu Mar 16 03:31:43 2023 +0000
+++ b/netwerk/protocol/http/nsHttpChannel.cpp	Thu Mar 16 12:43:08 2023 +0000
@@ -8193,8 +8193,7 @@
         socketTransport->GetSelfAddr(&mSelfAddr);
         socketTransport->GetPeerAddr(&mPeerAddr);
         socketTransport->ResolvedByTRR(&isTrr);
-        MOZ_ALWAYS_SUCCEEDS(
-            socketTransport->GetEffectiveTRRMode(&mEffectiveTRRMode));
+        socketTransport->GetEffectiveTRRMode(&mEffectiveTRRMode);
         socketTransport->GetEchConfigUsed(&echConfigUsed);
       }
     }
